mender-convert

```diff
@@ -64,7 +64,6 @@ Examples:
                 --raw-disk-image <raw_disk_image_path>
                 [--mender-disk-image <mender_image_name>]
                 --device-type <beaglebone | raspberrypi3>
-                --mender-client <mender_binary_path>
```

Comment:
The same here - just add brackets for optional argument.

Reply:
Here yes! It makes sense to leave it with brackets :+1: